website/content/docs/agent/config/config-files.mdx

```diff
@@ -870,9 +870,9 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
       [`acl.tokens.initial_management`](#acl_tokens_initial_management).**
 
     - `default` ((#acl_tokens_default)) - When provided, the agent will
-      use this token when making requests to the Consul servers. Clients can override
-      this token on a per-request basis by providing the "?token" query parameter.
-      When not provided, the empty token, which maps to the 'anonymous' ACL token,
+      use this token when making requests to the Consul servers.
+      Clients can [override this token on a per-request basis](/api-docs#authentication).
```

Comment:
```suggestion
      Clients can provide an alternate token in the authorization header to override this token on a per-request basis. Refer to [Authentication](/api-docs#authentication) for details.
```
I think we can provide a little more specificity and context for what the user can expect when they visit the link.